sync_skeleton.sh

```diff
@@ -30,65 +59,90 @@ PAYLOAD=$(cat <<EOF
 EOF
 )
 
+default_files='
+  .gitignore
+  .yamllint
+  CONTRIBUTING.md
```

Comment:
No license?

Reply:
Good catch.

Reply:
LICENSE cannot be synced as it has different "Copyright" line in different repos.

Reply:
We solved this in Prometheus by moving the Copyright to a `NOTICE` file.

Reply:
I get it in the case of prometheus as it uses Apache License 2.0 and `NOTICE` has a list of libraries used in the project. In the case of ansible roles we have MIT License and there are no libraries consumed by roles so `NOTICE` file would contain only a few lines in each repo.

That said I am no lawyer, but in the case of MIT license, I don't think we should move Copyright out of `LICENSE` file.

Reply:
I've removed the LICENSE file from the sync. We can figure out LICENSE/NOTICE stuff later.

Reply:
@paulfantom Sounds fine to me.